README.md

```diff
@@ -50,3 +49,9 @@ Submit issues and contributions:
 - You can also build the extension `.vsix` yourself by running [`vsce package`](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#packaging-extensions) from `dev/`. Refer to the [`Jenkinsfile`](https://github.com/eclipse/codewind-vscode/blob/master/Jenkinsfile) to see the exact steps the build runs.
 - The extension bundles dependency executables. These are gitignored, but should be kept up-to-date on your local system with the same versions used in the `Jenkinsfile` `parameters` section. Run `dev/bin/pull.sh` to download the dependencies. Also see `dev/bin/README.txt`.
 - The [`prebuild`](https://github.com/eclipse/codewind-vscode/blob/master/dev/prebuild.js) script is used in the CI builds to build separate versions of the extension for VS Code and Theia, since each of those has some commands that the other does not. It deletes inapplicable commands from the `package.json`, and does not modify any ts/js code. Run this before `vsce package` to get a closer-to-production build, but be ready to revert the changes.
+
+## Building Codewind from the source
+1. Clone the [`codewind`](https://github.com/eclipse/codewind) repository.
+2. Clone the `codewind-vscode` repo.
+3. Run the `./script/build.sh` script to run the Codewind build, or run the `./run.sh` script to build and start Codewind.
```

Comment:
one more clarifying detail would be to 'run `codewind/script/build.sh`' or `codewind/script/run.sh`

so they know to look under codewind, not under codewind-vscode, for those scripts